.github/workflows/run-validation.yml

```diff
@@ -0,0 +1,67 @@
+name: Run validation with RoboValidate
```

Comment:
Excellent landing pages, but the titles do not need to be all caps http://digitalgov.lndo.site/admin/content?title=&type=landing_page&status=All